content/en/blog/_posts/2023-01-17-confidential-kubernetes.md

```diff
@@ -0,0 +1,152 @@
+
+Confidential virtual machines based on AMD SEV-SNP require no changes to the executed program and operating system and are a lot easier to benchmark. A [benchmark from Azure and AMD shows](https://community.amd.com/t5/business/microsoft-azure-confidential-computing-powered-by-3rd-gen-epyc/ba-p/497796) that SEV-SNP VM overhead is <10%, sometimes as low as 2%.
+
+Although there is a performance overhead, it should be low enough to enable real-world workloads to run in these protected environments and improve the security and privacy of our data.
+
+## Confidential Computing compared to FHE, ZKP, and MPC
+
+Fully Homomorphic Encryption (FHE), Zero Knowledge Proof/Protocol (ZKP), and Multi-Party Computations (MPC) are all a form of encryption or cryptographic protocols that offer similar security guarantees to confidential computing but do not require hardware support.
+
+Fully (also partially and somewhat) homomorphic encryption allows one to perform computations, such as addition or multiplication, on encrypted data. This provides the property of encryption in use but does not provide integrity protection or attestation.
+
+Zero Knowledge Proofs or Protocols are a privacy-preserving technique (PPT) that allows one party to prove facts about their data without revealing anything else about the data. ZKP can be used instead of or in addition to confidential computing to protect the privacy of the involved parties and their data. Similarly, Multi-Party Computation enables multiple parties to work together on a computation, i.e, each party provides their data to the result, without leaking their data to any of the other parties.
+
+## Use Cases of Confidential Computing
+
+The presented confidential computing platforms show that both the isolation of a single process and therefore minimization of the trusted computing base, and the isolation of a full virtual machine are possible. This already enabled a lot of interesting and secure projects to emerge:
+
+### Confidential Containers
+
+The [Confidential Containers project](https://github.com/confidential-containers) enables users to run a container inside a confidential context. It provides an abstraction layer so that users do not need to interface directly with the confidential computing hardware.
+
+Confidential Containers is a [CNCF sandbox project](https://www.redhat.com/en/blog/what-confidential-containers-project).
+
+> TODO: @Tobin/Mikko (or other CoCo folks) can you expand on this chapter?
```

Comment:
Yes

Reply:
Awesome - thanks so much for providing this @mkbhanda!